packages/pycryptodomex/meta.yaml

```diff
@@ -1,12 +1,14 @@
 package:
   name: pycryptodomex
   version: 3.15.0
+  top-level:
+    - Cryptodome
```

Comment:
Intestingly, this file was named wrong (`meta.yml`), so this package was not built.

Reply:
It seems like the build is faling, disabled for now.